.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
```

Comment:
Looks like this is some sort of JetBrains thing?

I am cool with it being in there but could you just add a comment with some info and a link to docs on the file (https://www.jetbrains.com/help/idea/editorconfig.html) just to make it obvious what it is being used for.

Reply:
EditorConfig is intended to help standardize code style configurations across editors. (It overrides the JetBrains IntelliJ configurations if those are present.)

Support for EditorConfig comes built-in by default in lots of editors, including most of the IntelliJ-based ones as well as Microsoft's full Visual Studio (not vscode). There are plugins offered for many other editors as well, including vscode.

For more information about it, you can see https://editorconfig.org/ and I'll add a link to that at the top of the file as well.